dactyl-config.yml

```diff
@@ -124,341 +124,344 @@ targets:
             "img/demurrage-currency-code-format.png": https://cdn.ripple.com/wp-content/uploads/2017/11/demurrage-currency-code-format.png
             "img/xrp-ledger-network-ledger-store-and-shard-store.png": https://cdn.ripple.com/wp-content/uploads/2018/02/xrp-ledger-network-ledger-store-and-shard-store1.png
 
-    -   name: rippled-setup
-        display_name: rippled Setup Guide
-        link_subs:
-            "reference-rippled.html": https://ripple.com/build/rippled-apis/
-
 pages:
+  # I. Introduction ------------------------------------------------------------
+
     -   md: concept-xrp-ledger-intro.md
         category: Introduction
         targets:
             - local
             - ripple.com
 
-# References are exhaustive lists of commands and options
-    -   name: RippleAPI
-        category: References
-        html: reference-rippleapi.html
-        # Currently this is the only page that's fetched remotely.
-        md: https://raw.githubusercontent.com/ripple/ripple-lib/e6d71471e26ca253e762fc7034859951b5d31c47/docs/index.md
-        filters:
-            - remove_doctoc
-            - add_version
+    # "XRP Ledger Ecosystem Overview" goes here
+
+  # II. Technical Overview -----------------------------------------------------
+
+    -   md: concept-consensus.md
+        category: Technical Overview
         targets:
             - local
             - ripple.com
 
-    -   name: rippled
-        category: References
-        html: reference-rippled.html
-        md: reference-rippled.md
+    -   md: concept-reaching-consensus.md
+        category: Technical Overview
         targets:
             - local
             - ripple.com
 
-    -   name: Transaction Format
-        category: References
-        html: reference-transaction-format.html
-        md: reference-transaction-format.md
+    -   md: concept-transactions.md
+        category: Technical Overview
```

Comment:
Sort of a nit, but why call this "Technical Overview"? It's in the technical documentation site/section so I think the "technical" could be assumed, right? 

Along similar lines, I think the other category names could be shortened. That would have a couple of benefits: improved brevity and clarity, decreased ambiguity (especially w/r/t "XRP Tasks"), and more usable space on the landing page and sidebar. Therewith, here are my suggestions for new names:

1. Technical Overview --> Overview (I would also be OK with XRP Ledger Overview; that's wordier but more relevant)  
2. Reference Material --> Reference ("Reference Material" sounds like the dustiest section of the library)
3. About XRP (or XRP Overview to be parallel with the XRP Ledger overview)
4. The rippled Server --> Using the rippled Server OR Working with rippled (side note: I can understand why the `rippled-reference.html` is in this section, but, as a user, I'd probably expect that to be in the...Reference section. This might be a case for listing the same topic twice?)
5. XRP Tasks --> Working with XRP OR Using XRP
6. The Exchange Business Model -->  Exchanges
7. The Gateway Business Model --> Gateways
8. Issued Currencies --> Working with Issued Currencies
9.  API Tools (I'm OK with this)